### DISCUSSION TITLE
Sequence diagrams don't render on the Github mobile app

### DISCUSSION THREAD

**Discussion by rishitank** [community/community#128496] [Mobile]
When viewing a pull request on the Github mobile app, sequence diagrams only show the code and are not rendered / interactive. Are there plans to support this?

**Comment by Iron-Ham** [comment]
@rishitank Thank you for reporting this! We are internally tracking the addition of mermaid diagram support for our mobile apps, and I've added your feedback to the open issue for us to look into.
